workflows/readsimulator/main.nf

```diff
@@ -35,17 +35,16 @@ ch_multiqc_custom_methods_description = params.multiqc_methods_description ? fil
 //
 // MODULE: Local modules
 //
-include { MERGE_FASTAS            } from '../modules/local/custom/merge_fastas/main'
-include { INSILICOSEQ_GENERATE    } from '../modules/local/insilicoseq/generate/main'
-include { CREATE_SAMPLESHEET      } from '../modules/local/custom/create_samplesheet/main'
-include { MERGE_SAMPLESHEETS      } from '../modules/local/custom/merge_samplesheets/main'
-include { WGSIM                   } from '../modules/local/wgsim/main'
+include { INSILICOSEQ_GENERATE    } from '../../modules/local/insilicoseq/generate/main'       // TODO: Add module to nf-core/modules
+include { CREATE_SAMPLESHEET      } from '../../modules/local/custom/create_samplesheet/main'
+include { MERGE_SAMPLESHEETS      } from '../../modules/local/custom/merge_samplesheets/main'
+include { WGSIM                   } from '../../modules/local/wgsim/main'                      // TODO: Add module to nf-core/modules
 
 //
 // SUBWORKFLOW: Consisting of a mix of local and nf-core/modules
 //
-include { AMPLICON_WORKFLOW       } from '../subworkflows/local/amplicon_workflow'
-include { TARGET_CAPTURE_WORKFLOW } from '../subworkflows/local/target_capture_workflow'
+include { AMPLICON_WORKFLOW       } from '../../subworkflows/local/amplicon_workflow'
+include { TARGET_CAPTURE_WORKFLOW } from '../../subworkflows/local/target_capture_workflow'
 
```

Comment:
Does this work? It works for me with one `../`

Reply:
I reorganized the workflow directory ( workflows > readsimulator > main.nf ) because it has been recommended to me recently. I think the plan is that this structure will help in chaining pipelines by allowing installation of multiple workflows.